app/containers/Forms/SummaryFormFieldTemplate.tsx

```diff
@@ -67,29 +67,41 @@ const FormFieldTemplate: React.FunctionComponent<FieldTemplateProps> = ({
           display: none;
         }
         .summary-item {
-          display: inline-block;
-          width: 70%;
+          flex-basis: 70%;
+          // Allows the item to grow to 100% when summary-label is hidden
+          flex-grow: 1;
+          display: inline-flex;
+          align-items: flex-start;
           border: 1px solid;
+          border-color: #aaa;
           margin-bottom: -1px;
           margin-left: -1px;
-          padding: 5px;
-          border-color: #aaa;
+        }
+        .summary-item > * {
           padding: 5px 10px;
         }
+
+        .summary-item > source-field-heading {
+          padding: 0;
+        }
         .emission .form-row .summary-formgroup:nth-of-type(2n) {
           margin-top: 50px;
           margin-bottom: 20px;
         }
-        .emission h3 {
-          margin: 0;
-        }
+
         .form-row .summary-formgroup:nth-of-type(2n) {
           background-color: #f1f1f1;
         }
+        .emission .summary-label {
+          border: none;
+        }
         .emission .summary-item {
-          border: 0;
-          padding: 20px 20px 10px 0;
+          border: none;
+          display: inline-flex;
+          justify-content: space-around;
+          align-items: flex-start;
         }
+
         .summary-card .card-body {
           padding-top: 0;
           padding-bottom: 50px;
```

Comment:
Some change in this file has affected the size of the triangle exlamation mark icons that appear next to required property errors. Best starting point I can give is that sometimes setting flex properties can do funky unexpected things to the sizing of elements - check `flex-basis`, `flex-grow`, `flex-shrink` and similar:
<img width="1440" alt="Screen Shot 2020-07-23 at 12 38 20 PM" src="https://user-images.githubusercontent.com/5522075/88331740-4fb29400-cce2-11ea-973f-e3496dcd556d.png">
<img width="1439" alt="Screen Shot 2020-07-23 at 12 41 22 PM" src="https://user-images.githubusercontent.com/5522075/88331748-5214ee00-cce2-11ea-8f43-073c0310fa2d.png">

Reply:
Also: 
<img width="1440" alt="Screen Shot 2020-07-23 at 12 48 52 PM" src="https://user-images.githubusercontent.com/5522075/88332240-14649500-cce3-11ea-843b-6b46f0c5ebfc.png">

Reply:
Looks good on Firefox. I'll add some happo screenshots with validation errors to make sure we're checking that CSS across browser

Reply:
(actually on Firefox, the warning signs are not displayed at all)